README.md

```diff
@@ -125,6 +125,155 @@ record = client.write(record_type, data)
+```python
+# Starts search from the first page of results
+after_index = 0
+
+# Amount of records to returned, limiting if more are available. Defaults to 50 to a maximum of 1000.
+count = 50
+
+# Include only records written by the client (False), or search other writer ids (True)
+include_all_writers = False
+
+# Include data when returning records
+include_data = False
+```
+
+Under Search Params there are these defaults:
+```python
+# Conditional OR when searching upon all fields within this Param object
+condition = "OR" # options: "OR"|"AND"
+
+# Exactly matches the search fields provided
+strategy = "EXACT" # options "EXACT"|"FUZZY"|"WILDCARD"|"REGEXP"
+```
+
+Under Search Range there is this default:
+```python
+# Search records based on when they were created of last modified
+key = "CREATED" # options: "CREATED"|"MODIFIED"
+
+# Default time provided is set to UTC.
+zone = "UTC" # options are "PST":"-08:00", "MST":"-07:00", "CST":"-06:00", "EST":"-05:00", "UTC":"+00:00"
+
+# Zone offset is None by default, but if provided it will override the default zone parameter
+zone_offset = None
+```
+
+Since python datetime objects are zone agnostic, provide the proper timezone offset from UTC
+in zone_offset to search properly.
+
+### Advanced Matching Strategies
+
+Search offers advanced queries that provide more flexibility than the default matching strategy of `EXACT`. The four options are `EXACT`, `FUZZY`, `WILDCARD`, and `REGEXP`. Depending on the matching strategy and the fields provided, these searches may be slower. 
+
+To mirror some of the above queries with these matching strategies we get:
+```python
+# e3db setup...
+
+# fuzzy
+fuzz_query = Search().Match(strategy="FUZZY", record_type=["season"], values=["bummer"])
+
+# wildcard
+wild_query = Search().Match(strategy="WILDCARD", record_type=["season"], values=["su??er"])
+
+# regexp
+regxp_query = Search().Match(strategy="REGEXP", record_type=["season"], values=["sum.*"])
+```
+
+### Paging
+
+The construction of the Search object offers a number of options for paging through your results: namely `after_index` and `count`.
+
+To page through a large number of results, you can loop like this:
+```python
+# e3db setup...
+# limit number of results returned to 10 at a time
+query = Search(count=10).Match(record_type=["many_results"])
+results = client.Search(query)
+
+# Number of results in e3db
+total_results = results.total_results
+# Return records from this point onwards in next query
+
+while results.after_index:
+    query.after_index = results.after_index
+    results = client.Search(query)
+    do_something(results) # process results
+```
+
+The `after_index` returned from a query will be 0 if there are no more records to return. `total_results` represents the total number of records in e3db that match the executed query. 
+
+See [the integration tests](https://github.com/tozny/e3db-python/blob/master/e3db/tests/test_search_integration.py) for more examples.
```

Comment:
You should be able to reference relative url to the test_search_integration.py file, instead of having to include the entire github url
https://help.github.com/articles/about-readmes/#relative-links-and-image-paths-in-readme-files